### PR TITLE
tweak travel and explore speed and angle params

### PR DESCRIPTION
**Pull request by theodu33160**
What do you think about these params?

### CODE REVIEW

File: src/operations/land_operation.cpp [hunk 1]
```diff
@@ -9,7 +9,7 @@
 LandOperation::LandOperation() : Operation(OperationIdentifier::LAND, true, true) {}
 
 bool LandOperation::isBelowThreshold() const {
-    return getCurrentPose().pose.position.z < 0.40 &&
+    return getCurrentPose().pose.position.z < 0.05 &&
```
Comment (theodu33160) [inline src/operations/land_operation.cpp:12]: This had been changed by mistake. I just put it back to previous value

**Comment by h-brenne** [comment]
What do we do if the wind requires angles above max angle? Maybe we should rely more on max ground speed, allowing higher max angle(That still has enough vertical component to lift the drone at high throttle)?

**Comment by theodu33160** [comment]
> What do we do if the wind requires angles above max angle? Maybe we should rely more on max ground speed, allowing higher max angle(That still has enough vertical component to lift the drone at high throttle)?

If the desired angle is above the max_angle, then ardupilot with constrain our desire and set the desired angle to max angle. I think that we want to unsure that the the drone has enough vertical lift, so this is the max angle that should be set. The velocity depends on other parameters such as wind and makes it unreliable when it comes to the amount of vertical lift we get.

**Comment by h-brenne** [comment]
I was thinking about the low max angle in the exploration phase. Wouldn't it be an option to always have a high max angle, and rather use max speed to limit the velocity? Then the drone can compensate for wind, center of mass etc. if necessary.

**Comment by theodu33160** [comment]
Good remark. the problem is that when we set a low speed with high max angle, then the drone leans a lot to reach the desired speed in no time. I'll see later when I'll get some free time if we can set a maximum acceleration in stead

**Pull request merged**